examples/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+cmake_minimum_required(VERSION 3.11)
+
+get_filename_component(ImGui_SRCDIR "${CMAKE_CURRENT_LIST_DIR}" DIRECTORY)
+
+include("${CMAKE_CURRENT_LIST_DIR}/ImGuiModule.cmake")
+
+set(ImGui_OPTIONS)
+
+imgui_option(EXAMPLES "Dear ImGui example applications" ON)
+imgui_option(BINDINGS "Dear ImGui example bindings" ON)
+imgui_option(3THPARTY "Dear ImGui example dependencies" ON)
```

Comment:
Should be 3RDPARTY or even better - THIRDPARTY. No need to shorten the word.

Reply:
Even more:

EXAMPLES -> BUILD_EXAMPLES
BINDINGS -> BUILD_BINDINGS
3THPARTY -> USE_SYSTEM_DEPS (will tell user that dependencies that are installed on the system will be used. Much clearer to end-users)